internal/msgqueue/v1/rabbitmq/rabbitmq.go

```diff
@@ -610,6 +611,10 @@ func (t *MessageQueueImpl) subscribe(
 				if err := preAck(msg); err != nil {
 					t.l.Error().Msgf("error in pre-ack on msg %s: %v", msg.ID, err)
 
+					if err == metered.ErrResourceExhausted {
```

Comment:
This probably isn't the right place for this -- I would put it in the tasks controller. The MQ shouldn't concern itself with specific error states and we have two MQ implementations, so this is a bit confusing